src/components/SearchList.js

```diff
@@ -1,23 +1,33 @@
-import React, {Component} from 'react';
-import {Platform, Text} from 'react-native';
-import {Button} from 'react-native-elements';
+// 3rd party libraries - core
+import React from 'react';
+import {View, Text, StyleSheet} from 'react-native';
+import {Card} from 'react-native-elements';
+// 3rd party libraries - additional
 
+// styles and language
 
-import {CardSection, Card} from './common/index';
+// our components - core
+// our components - additional
 
-class SearchList extends Component {
+const SearchList = ({sites}) => {
```

Comment:
Doesn't need to be a class-based component. Both this and the `SearchMap` can just be functional components that take the relevant props and display them.